orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -5874,25 +5892,26 @@ def generate_inputs(batch_size, max_seq_length, vocab_size):
     else:
         assert len([node.op_type for node in training_model.graph.node if node.op_type == "ShrunkenGather"]) == 1
     gathergrad_node = [node for node in training_model.graph.node if node.op_type == "GatherGrad"][0]
-    if test_op == "Add":
+    def find_input_node_type(model, arg):
+        result = []
+        for node in model.graph.node:
+            for output in node.output:
```

Comment:
ifr arg in node.output:
  result.append(node)

Reply:
done